packages/core/src/utils.ts

```diff
@@ -714,3 +715,55 @@ function toPreciseInteger(a: number, precision?: number): number {
   }
   return Math.round(a * Math.pow(10, precision));
 }
+
+/**
+ * Returns a human-readable string for a FHIR Range datatype, taking into account comparators and one-sided ranges
+ * @param range A FHIR Range element
+ * @returns A human-readable string representation of the Range
+ */
+export function formatRangeString(range: Range | undefined): string {
```

Comment:
This has high duplication with code in `RangeDisplay`.  Should be able to squash into a single function: https://github.com/medplum/medplum/blob/main/packages/react/src/RangeDisplay/RangeDisplay.tsx

Reply:
Ok, I think this is a merege issue. I thought I had _moved_ the function, but somehow I duplicated. Will resolve